exasol_integration_test_docker_environment/testing/exaslct_test_environment.py

```diff
@@ -64,7 +64,7 @@ class ExaslctTestEnvironment:
 
     def __init__(
         self, test_object, executable="./exaslct", clean_images_at_close=True, name=None
```

Comment:
are the missing types intended

Reply:
Yes, will be fixed in https://github.com/exasol/integration-test-docker-environment/issues/453